services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dto.{ FileContent, ValidateError }
+import cromwell.pipeline.womtool.WomTool
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class ProjectFileService(womTool: WomTool)(implicit executionContext: ExecutionContext) {
+
+  def validateFile(fileContent: FileContent): Future[Either[ValidateError, Unit]] =
+    Future(womTool.validate(fileContent.content)).map {
+      case Left(value) => Left(ValidateError(value.toList.mkString("\n")))
```

Comment:
No need in this `.toList` method.
Add the following imports
```
import cats.instances.string._
import cats.syntax.foldable._
```
And use `mkString_` method.